index.html

```diff
@@ -189,9 +186,37 @@ <h2 class=" connect">LET'S CONNECT</h2>
                 <a href="#" class="dropdown-btn"><div>Skills</div><img src="images/arrow.png"></a>
             </li>
           </ul>
-       
+        </div>
     </section>
-    
+    <section class="contact-me">
+    
+        <div class="form-header">
+        <h2 class="form-heading"> Contact me </h2>
+ 
+        <p class="pg-contact">If you have an application you are interested in developing, a feature that you
+          need built or a project that needs coding. 
+         I’d love to help with it!
+         </p>
+         </div>
+         <form action="https://formspree.io/f/mqkvkgdo" method="POST" class="contact-me-form">
+         <ul class="form-info">
+         <li>     
+            <!-- <label for="name" value="name">Name:</label> -->
+             <input type="text" id="name" value="Lorem Ipsum" name="name" required placeholder="enter your name" maxlength="30"/></li>
+         <li>
+             <!-- <label for="e-mail" value="e-mail">E-mail:</label> -->
+                 <input type="email" id="e-mail" value="loremipsum@gmail.com" name="email" required placeholder="enter your email"/> </li>
+          <li>
+                 <!-- <label for="message" value="message" >Message</label><br> -->
+                 <textarea id="message" maxlength="500" name="message"  >Enter Your message here...</textarea> </li>
```

Comment:
- [ ] Great job on implementing the contact form for your portfolio as specified in the project requirement! 👏 However, there are some minor issues 🐞 that we need to fix before moving on to the next milestone. To do so kindly consider the below suggestions:

   - Kindly consider adding the `required` attribute to your `textarea` element, as specified in the project requirements. This attribute ensures that users must fill in the `textarea` field before they can submit the form. By making this field required, you enhance the user experience by prompting them to provide the necessary information.